roles/keyboardwalk.role

```diff
@@ -29,8 +28,9 @@ NUCLEAR_ROLE(
     # Behaviour
     behaviour::Controller
     behaviour::skills::Stand
-    #behaviour::skills::FallingRelax
+    behaviour::skills::FallingRelax
     behaviour::skills::Getup
+    behaviour::skills::KickScript
```

Comment:
Has this been tested?

Reply:
Not properly tested. Removed for this PR